CoolNotifications/autorun/client/CoolNotifs.lua

```diff
@@ -0,0 +1,97 @@
+surface.CreateFont( "NoticeFont", {
+	font = "BITSTREAM VERA SANS", --Use the font-name which is shown to you by your operating system Font Viewer, not the file name
+	size = 16.2,
+	weight = 2000,
+	antialias = true
+} )
+
+
+local PANEL = {}
+
+function PANEL:Init()
+	self:SetSize(256, 30)
+	self:SetContentAlignment(5)
+	self:SetExpensiveShadow(1, Color(0, 0, 0, 150))
+	self:SetFont("NoticeFont")
+	self:SetTextColor(color_white)
+end
+
+function PANEL:Paint(w, h)
+
+	surface.SetDrawColor(230, 230, 230, 10)
+	surface.DrawRect(0, 0, w, h)
+
+	if (self.start) then
+		local w2 = math.TimeFraction(self.start, self.endTime, CurTime()) * w
+
+		surface.SetDrawColor(Color( 0, 0, 0, 230 ))
+		surface.DrawRect(0, 0, w, h)
+		surface.SetDrawColor(Color(255,255,255))
+		surface.DrawRect(w2, h-2, w - w2, 2)
+	end
+
+	surface.SetDrawColor(0, 0, 0, 45)
+	surface.DrawOutlinedRect(0, 0, w, h)
+end
+
+vgui.Register("DNotice", PANEL, "DLabel")
+
+notices = notices or {}
+
+function CoolNotify(message,delay)
+	local notice = vgui.Create("DNotice")
+	local i = #notices+1
+	local scrW = ScrW()
+
+	notices[i] = notice
+	
+	notice:SetText(message)
+	notice:SetPos(ScrW(), ScrH() - (i - 1) * (notice:GetTall() + 4 	) + 4)
+	notice:SizeToContentsX()
+	notice:SetWide(notice:GetWide() + 64)
+	notice.start = CurTime() + 0.25
+	notice.endTime = CurTime() + delay
+	notice.id = i	
+	notice.OnRemove = function() 
+		notices[notice.id] = nil
```

Comment:
I think you should replace this with `table.remove(notices,notice.id)` but try it out first